t/e2e/transactions/transactions_test.go

```diff
@@ -361,33 +184,18 @@ func (s *TransactionsTestSuite) TestSendEther() {
 
 	EnsureNodeSync(s.Backend.StatusNode().EnsureSync)
 
-	// create an account
-	sampleAddress, _, _, err := s.Backend.AccountManager().CreateAccount(TestConfig.Account1.Password)
+	err := s.Backend.AccountManager().SelectAccount(TestConfig.Account1.Address, TestConfig.Account1.Password)
 	s.NoError(err)
 
-	completeQueuedTransaction := make(chan struct{})
-
-	// replace transaction notification handler
-	var signRequestResult []byte
-	s.setDefaultNodeNotificationHandler(&signRequestResult, sampleAddress, completeQueuedTransaction, nil)
-
-	// this call blocks, up until Complete Transaction is called
-	txHashCheck, err := s.Backend.SendTransaction(context.TODO(), transactions.SendTxArgs{
+	result := s.Backend.SendTransaction(transactions.SendTxArgs{
 		From:  account.FromAddress(TestConfig.Account1.Address),
 		To:    account.ToAddress(TestConfig.Account2.Address),
 		Value: (*hexutil.Big)(big.NewInt(1000000000000)),
-	})
-	s.NoError(err, "cannot send transaction")
+	}, TestConfig.Account1.Password)
 
-	select {
-	case <-completeQueuedTransaction:
-	case <-time.After(2 * time.Minute):
-		s.FailNow("completing transaction timed out")
-	}
+	s.NoError(result.Error, "cannot send transaction")
 
-	s.Equal(txHashCheck.Bytes(), signRequestResult, "transaction hash returned from SendTransaction is invalid")
-	s.False(reflect.DeepEqual(txHashCheck, gethcommon.Hash{}), "transaction was never queued or completed")
-	s.Zero(s.Backend.PendingSignRequests().Count(), "tx queue must be empty at this point")
+	s.False(reflect.DeepEqual(result.Response.Hash(), gethcommon.Hash{}), "transaction was never queued or completed")
```

Comment:
`"transaction was never queued or completed"` is inaccurate now. We can remove it.